### PR TITLE
Fix an issue missing page height

### PR DESCRIPTION
**Pull request by tsub**
Fixed https://github.com/github/personal-website/issues/94

### When there are little content on the page

image

### When there are many page contents

Page height is not enough.

image

### When there are little content on the page

image

### When there are many page contents

The height of the page also increases according to the height of the content.

image

### CODE REVIEW

File: assets/styles.scss [hunk 1]
```diff
@@ -37,3 +37,7 @@
     padding-left: 16px;
   }
 }
+
+.min-height-full {
+  min-height: 100vh;
```
Comment (tsub) [inline assets/styles.scss]: `min-height: 100%;` did not work as expected.

image

Comment (tsub) [inline assets/styles.scss]: `vh` works fine with most browsers.

> image
>
> https://caniuse.com/#search=vh

**Comment by aadibajpai** [comment]
I just had to revert #58 to make it work.

**Comment by tsub** [comment]
I reverted a merge commit c934c47 of #58
Then I made a commit 8218492 to fix an issues.

@aadibajpai Was this correct?

**Comment by aadibajpai** [comment]
It works so I guess, yep 👍

**Pull request merged**